timeline/style.css

```diff
@@ -0,0 +1,84 @@
+.timeline-line:after {
+  content: "";
+  position: absolute;
+  width: 2px;
+  height: 100%;
+  background: #dddddd;
+  z-index: 1;
+  left: 16px;
+}
+.space-marker {
+  z-index: 9
+}
+.marker {
+  margin: 0 auto;
+  margin-top: 10px;
+}
+.calendar-date {
+  min-height: 120px;
+}
+.calendar-date span {
+  font-size: 4em
+}
+.group-user-photo {
+  -webkit-transform: translateX(10px);
+          transform: translateX(10px);
+}
+.user-photo {
```

Comment:
Since this feature is only for the `timeline`, start the class with `.timeline .user-photo`